docs/release-notes/2.4/2.4.27.md

```diff
@@ -0,0 +1,37 @@
+# Percona XtraBackup 2.4.27 (2022-12-7)
+
+<style> table { width=50%'; font-family: Chivo, Colfax, Franziska, Helvetica, Arial, sans-serif; } table td { border: 0px; padding: 8px; } tr:nth-child(even){ background-color:#f5f5f5 } </style>
+| Release date | December 7, 2022 |
+|---|---|
+| Install instructions | [Install Percona XtraBackup 2.4](https://docs.percona.com/percona-xtrabackup/2.4/installation.html) |
+| Download this version | [Percona XtraBackup 2.4](https://www.percona.com/downloads/Percona-XtraBackup-2.4/LATEST/) |
+
+[Percona XtraBackup for MySQL Databases](https://www.percona.com/software/mysql-database/percona-xtrabackup) enables MySQL backups without blocking user queries. Percona XtraBackup is ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, Percona XtraBackup drives down backup costs while providing unique features for MySQL backups.
+
+Percona XtraBackup 2.4 does not support making backups of databases created in MySQL 8.0, Percona Server for MySQL 8.0, or Percona XtraDB Cluster 8.0. Download [Percona XtraBackup 8.0](https://www.percona.com/downloads/Percona-XtraBackup-LATEST/#) to make backups for these versions.
+
+## Release highlights
+
+This release contains fixes for backup up Percona XtraDB Cluster and an xbcloud fix.
+
+## Bugs fixed
+
+* [PXB-2809](https://jira.percona.com/browse/PXB-2809): `wsrep_sync_wait<>0` caused a `Lock wait timeout exceeded` error when issued against Percona XtraDB Cluster (Thanks to Frank Well for providing the initial patch).
+
+* [PXB-2840](https://jira.percona.com/browse/PXB-2840): Xbcloud did not have a required set location constraint when creating a bucket on a non-default endpoint.
```

Comment:
for to mention, this was also a contribution, please add

Thanks to Jason Zareski for the patch.

Reply:
Done.